cargo-crev/src/doc/getting_started.md

```diff
@@ -306,7 +306,7 @@ all the ids you trust.
 $ cargo crev query id trusted
 FYlr8YoYGVvDwHQxqEIs89reKKDy-oWisoO0qXXEfHE medium https://github.com/dpc/crev-proofs
 YWfa4SGgcW87fIT88uCkkrsRgIbWiGOOYmBbA1AtnKA low    https://github.com/oherrala/crev-proofs
```

Comment:
What about these ones?

Reply:
That dpc is the one referred to as user `dpc` in the docs, I believe